addons/editor-theme3/theme3.css

```diff
@@ -29,3 +29,31 @@
   fill: var(--editorTheme3-commentText);
   color: var(--editorTheme3-commentText);
 }
+
+.sa-colormode-submenu li:nth-child(1) div,
+.sa-colormode-submenu li:nth-child(2) div {
+  filter: blur(1px) grayscale(1);
```

Comment:
I personally prefer:
```suggestion
  filter: opacity(0.5) grayscale(1);
```
![Without blur effect](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/409b6830-d58e-4b46-bfc4-517f77ced076)

Reply:
That looks way better!